pkg/condition/condition.go

```diff
@@ -163,6 +166,15 @@ func (c *EvaluableCondition) Evaluate(contextMaps ...map[string]any) (Evaluation
 		return emptyEvaluationResult, fmt.Errorf("failed to construct condition partial vars: %v", err)
 	}
 
+	var missingParameters []string
```

Comment:
todo(@jon-whit): note this will produce duplicates if the partial bindings cannot be evaluated to produce a `bool`. The bit here https://github.com/openfga/openfga/pull/1108/files#diff-4e95dc26dacdca908d7eb173740544f27a74edf6474f9fbbeaa5461f57089fc7R187 will append to the list in this case and we'll get duplicates.